starter_project_mobile_2022/lib/blog_detail_murad/costants.dart/constants.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+const Color white = Colors.white;
+const Color black = Color.fromARGB(255, 0, 0, 0);
+const Color bgColorCommentDisplay = Color.fromRGBO(246, 246, 246, 1);
```

Comment:
Don't hesitate to write long and self-explanatory variable names such as;
`const Color backgroundColorCommentDisplay = Color.fromRGBO(246, 246, 246, 1);`